transfer.md

```diff
@@ -39,6 +39,7 @@ There are other equivalents for these tools
 | :---: | :---: |
 | `docker attach` | [`kpod exec`](./docs/kpod-attach.1.md) ***|
 | `docker build`  | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
+| `docker commit` | [`buildah commit`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-commit.md) |
```

Comment:
We should add a `kpod commit` command.

Reply:
maybe. should i change the PR in some way?

Reply:
No this is fine.  Just thinking outload